doc/changelog.md

```diff
@@ -34,7 +34,7 @@ Syntax: `- short text describing the change _(Your Name)_`
 - _()_
 - Migrate from Jest to Vitest, update Vite to v5, update Node to 20, .env should be .env.local _(Paul)_
 - Fix broken .env reading for scraper _(Paul)_
-- _()_
+- Add a dev env variable for react query when developing in offline env (e.g. plane) _(Paul)_
```

Comment:
What is the name of this env var?
In general: Try to keep env vars as minimal as possible, only add them when you really need to change it from time to time.